features/bootstrap/IsolatedProcessContext.php

```diff
@@ -69,4 +83,45 @@ public function iShouldSeeAnErrorAboutTheMissingAutoloader()
     {
         expect($this->process->getErrorOutput())->toMatch('/autoload/');
     }
+
+    /**
+     * @When I run phpspec
+     */
+    public function iRunPhpspec()
+    {
+        $process = new Process(
+            $this->buildPhpSpecCmd() . ' run'
+        );
+        $process->run();
+        $this->lastOutput = $process->getOutput();
+    }
+
+    /**
+     * @When I run phpspec with the :formatter formatter
+     */
+    public function iRunPhpspecWithThe($formatter)
+    {
+        $process = new Process(
+            $this->buildPhpSpecCmd() . " --format=$formatter run"
+        );
+        $process->run();
+        $this->lastOutput = $process->getOutput();
+    }
+
+    /**
+     * @Then I should see :message
+     */
+    public function iShouldSee($message)
+    {
+        expect(strpos($this->lastOutput, $message))->toNotBe(false);
+    }
+
+    /**
+     * @Given the isolated file :file contains:
```

Comment:
This should be in filesystemcontext and ideally integrated with the other existing methods